src/index.ts

```diff
@@ -71,6 +85,31 @@ export type UpsertResponse = CommitResponse;
 export type InsertCallback = CommitCallback;
 export type InsertResponse = CommitResponse;
 
+export interface LongRunningCallback {
+  (
+    err: ServiceError | null,
+    operation?: Operation,
+    apiResponse?: google.longrunning.IOperation
+  ): void;
+}
+export type LongRunningResponse = [Operation, google.longrunning.IOperation];
+
+export interface ExportEntitiesConfig
+  extends google.datastore.admin.v1.IExportEntitiesRequest {
+  bucket?: string | {name: string};
+  kinds?: string[];
+  namespaces?: string[];
+  gaxOptions?: CallOptions;
+}
+
+export interface ImportEntitiesConfig
+  extends google.datastore.admin.v1.IImportEntitiesRequest {
```

Comment:
Should not expose `projectId` here as well. As it is overridden [later](https://github.com/stephenplusplus/nodejs-datastore/blob/4325cd2f89d37c10813bb89de92f6a93d7ae5f77/src/request.ts#L921)
```suggestion
  extends Omit<google.datastore.admin.v1.IImportEntitiesRequest, 'projectId'> {
```